adzerk/transform.py

```diff
@@ -89,6 +89,7 @@ def to_collection(spocs):
         'title':     spocs[0]['collection_title'],
         'flight_id': spocs[0]['flight_id'],
         'sponsor':   spocs[0]['sponsor'],
+        'context':   __get_context(spocs[0]['sponsor']),
```

Comment:
I think I am missing something -- why do we need to store both the sponsor field as well as the context field, which now stores the "Sponsored by X" text? Or does the context just have "Sponsored by" and FX client creates the full text from context + sponsor? If so, the test is a little misleading.

Reply:
Offline chat with Mathijs clarified that the lack of localization means that we _can't_ construct that full "Sponsored by X" test with correct language support, which means we need to explicitly include that in context.